### PR TITLE
meat

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

map meat wall floor no rcd

## Changelog
 Not player facing.

### CODE REVIEW

File: code/modules/mapping/turfs.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/turf/open/floor/material/meat
+	custom_materials = list(/datum/material/meat = MINERAL_MATERIAL_AMOUNT * 0.25)
+
+/turf/closed/wall/material/meat
+	custom_materials = list(/datum/material/meat = MINERAL_MATERIAL_AMOUNT * 2)
+
+/atom
+	var/prevent_rcd_deconstruction = FALSE
```
Comment (GoldenAlpharex) [inline code/modules/mapping/turfs.dm:41]: Should be documented.

Comment (GoldenAlpharex) [reply code/modules/mapping/turfs.dm:41]: Also you don't seem to set it to true anywhere.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
A descriptive changelog entry would be nice.

File: code/modules/mapping/turfs.dm (hunk 1 above)
Comment (Gandalf2k15) [reply code/modules/mapping/turfs.dm:41]: It's for mapping, Duboris wants it for maps

Comment (Funce) [reply code/modules/mapping/turfs.dm:41]: He's shilling for sprites

Comment (Gandalf2k15) [reply code/modules/mapping/turfs.dm:41]: i'm shilling for you

**Pull request merged**